include/swift/AST/Decl.h

```diff
@@ -5647,6 +5647,8 @@ class ConstructorDecl : public AbstractFunctionDecl {
                   GenericParamList *GenericParams, 
                   DeclContext *Parent);
 
+  Identifier getName() const { return getFullName().getBaseIdentifier(); }
```

Comment:
I left it in for consistency. You can call `getName()` on all Decls that are guaranteed to be backed by Identifiers. But I don't think it's actually used. Long term goal would be to remove it once/if `Id_init` becomes a special name as well.